dev-java/open-test-reporting-events/open-test-reporting-events-0.1.0_pre1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source"
+MAVEN_ID="org.opentest4j.reporting:open-test-reporting-events:0.1.0-M1"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Language-agnostic test reporting format and tooling"
+HOMEPAGE="https://github.com/ota4j-team/open-test-reporting"
+SRC_URI="https://github.com/ota4j-team/open-test-reporting/archive/r${PV/_pre/-M}.tar.gz -> open-test-reporting-${PV}.tar.gz"
```

Comment:
Since `${PV/_pre/-M}` is reused in `S` below, I would introduce a variable for it. `MY_PN` could be helpful too:

```bash
MY_PN="open-test-reporting"
MY_PV="${PV/_pre/-M}"
MY_P="${MY_PN}-${MY_PV}"

SRC_URI="... -> ${MY_P}.tar.gz"

S="${WORKDIR}/${MY_P}/events"
```